bin/collator/src/local/service.rs

```diff
@@ -182,7 +187,10 @@ pub fn new_partial(
 }
 
 /// Builds a new service.
-pub fn start_node(config: Configuration) -> Result<TaskManager, ServiceError> {
+pub fn start_node(
+    config: Configuration,
+    tracing_config: EvmTracingConfig,
```

Comment:
Same comment as in command.rs file:

Please let's call this `evm_tracing_config` to make it more clear.
`tracing_config` sounds too generic.

Reply:
It should be, but let's specify it to EVM this time.